diff --git a/test/rekt/features/apiserversource/data_plane.go b/test/rekt/features/apiserversource/data_plane.go
index 037281409..1c7a734dd 100644
--- a/test/rekt/features/apiserversource/data_plane.go
+++ b/test/rekt/features/apiserversource/data_plane.go
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	exampleImage = "ko://knative.dev/eventing/test/test_images/print"
+	exampleImage = "registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-print"
 )
 
 func DataPlane_SinkTypes() *feature.FeatureSet {
