diff --git a/test/e2e/parallel_test.go b/test/e2e/parallel_test.go
index f5afe48c..e8fcd1cd 100644
--- a/test/e2e/parallel_test.go
+++ b/test/e2e/parallel_test.go
@@ -40,6 +40,7 @@ type branchConfig struct {
 }
 
 func TestFlowsParallel(t *testing.T) {
+	t.Skip("skipping")
 	const (
 		senderPodName = "e2e-parallel"
 	)
diff --git a/test/e2e/sequence_test.go b/test/e2e/sequence_test.go
index 0cbf7daf..9d220404 100644
--- a/test/e2e/sequence_test.go
+++ b/test/e2e/sequence_test.go
@@ -36,6 +36,7 @@ import (
 )
 
 func TestFlowsSequence(t *testing.T) {
+	t.Skip("skipping")
 	const (
 		sequenceName  = "e2e-sequence"
 		senderPodName = "e2e-sequence-sender-pod"
