diff --git a/test/e2e/source_sinkbinding_v1beta1_test.go b/test/e2e/source_sinkbinding_v1beta1_test.go
index 0df34b42e..086a5ba40 100644
--- a/test/e2e/source_sinkbinding_v1beta1_test.go
+++ b/test/e2e/source_sinkbinding_v1beta1_test.go
@@ -122,6 +122,7 @@ func TestSinkBindingV1Beta1Deployment(t *testing.T) {
 }
 
 func TestSinkBindingV1Beta1CronJob(t *testing.T) {
+	t.Skip("SRVKE-500: Skipping since we set bindings to inclusion")
 	const (
 		sinkBindingName = "e2e-sink-binding"
 		deploymentName  = "e2e-sink-binding-cronjob"
