diff --git a/test/e2e/channel_chain_test.go b/test/e2e/channel_chain_test.go
index 7bfd1927a..00c82b74b 100644
--- a/test/e2e/channel_chain_test.go
+++ b/test/e2e/channel_chain_test.go
@@ -32,6 +32,7 @@ EventSource ---> Channel ---> Subscriptions ---> Channel ---> Subscriptions --->
 
 */
 func TestChannelChainV1beta1(t *testing.T) {
+        t.Skip("upstream flake")
 	helpers.ChannelChainTestHelper(context.Background(), t, helpers.SubscriptionV1beta1, channelTestRunner)
 }
 
